README.md

```diff
@@ -5,7 +5,65 @@ This repository contains the code to generate various versions of the Datadog Ag
+|`DD_ENV`|The environment name where the Agent is running. The environment name is attached in-app to every metric, event, log, trace, and service check emitted by this Agent.|
+|`DD_APM_INSTRUMENTATION_ENABLED`|Automatically instrument all your application processes alongside Agent installation. Possible values are `host`, `docker` or `all` (`all` is both `host` and `docker`). The `docker` value checks if Docker is installed and sets `DD_NO_INSTALL` to `true`. If `DD_APM_INSTRUMENTATION_LANGUAGES` is not set, it enables all libraries: `java`, `js`, `python`, `dotnet` and `ruby`. `host` injection is not compatible with the `DD_NO_AGENT_INSTALL` option. Only supported on Agent v7. Not supported on SUSE.|
+|`DD_APM_INSTRUMENTATION_LANGUAGES`|Specify the APM library to be installed. Possible values are `all`, `java`, `js`, `python`, `dotnet` and `ruby`. Does not run the APM injection script, which is triggered by `DD_APM_INSTRUMENTATION_ENABLED`.|
+|`DD_APM_INSTRUMENTATION_NO_CONFIG_CHANGE`|Set to any value to pass the `--no-config-change` option to the APM host injection script.|
+|`DD_SYSTEM_PROBE_ENSURE_CONFIG`|Create the system probe configuration file from a template if it does not already exist.|
+|`DD_RUNTIME_SECURITY_CONFIG_ENABLED`|If set to `true`, ensure creation of security Agent and system probe configuration file (if they don't already exist), and enable Cloud Workload Security (CWS).|
+|`DD_COMPLIANCE_CONFIG_ENABLED`|If set to `true`, ensures the creation of a security Agent configuration file if one doesn't already exist, and enables Cloud Security Posture Management (CSPM).|
+|`DD_INSTALL_ONLY`|Set to any value to prevent starting the Agent after installation.|
+|`DD_NO_AGENT_INSTALL`|Do not install the Agent. Instead, installs package signature keys and creates configuration files if they don't already exist. Automatically set to `true` when `DD_APM_INSTRUMENTATION_ENABLED=docker`.|
+
+
+## Install script configuration options
+The install script also comes with its own configuration options for testing purpose.
+
+>[!WARNING]
+> These options are for Datadog internal use only.
+
+| Variable | Description|
+|-|-|
+|`DD_INSTRUMENTATION_TELEMETRY_ENABLED`|`true` if not set. When `false`, doesn't report telemetry to Datadog in case of a script issue.|
+|`DD_REPO_URL`|Domain name of the package S3 bucket to target. Default to `datadoghq.com`.|
+|`REPO_URL`|Deprecated, use `DD_REPO_URL` instead.|
+|`DD_RPM_REPO_GPGCHECK`|Turn on or off the `repo_gpgcheck` on RPM distributions. Possible values are `0` or `1`. Unless explicitely set, we turn off when `DD_REPO_URL` is set.|
+|`DD_AGENT_MAJOR_VERSION`|The Agent major version. Must be `6` or `7`|
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏ 
```suggestion
|`DD_AGENT_MAJOR_VERSION`|The Agent major version. Must be `6` or `7`.|
```